src/models/usuario.js

```diff
@@ -19,31 +20,39 @@ const ProfesionalSchema = Schema({
         type: String,
         required: true
     },
+    img: {
+        type: String
+    },
+    profesional: {
+        type: Boolean,
+        required: true
+    },
     sector: {
         type: String,
         required: true
     },
+    descripcion: {
+        type: String,
+        required: true
+    },
     cuentaBancariaIBAN: {
         type: String,
         required: true
     },
     titularCuenta: {
         type: String,
         required: true
-    },
-    img: {
-        type: String
-    },
-    eventosId: [{
-        type: Schema.Types.ObjectId,
-        ref: 'Evento'
-    }]
+    }
+});
 
+usuarioSchema.pre('save', function(next) {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7127635)
```suggestion
usuarioSchema.pre("save", function(next) {
```